dictionary/en.yaml

```
@@ -2141,7 +2133,7 @@ spi:
   id: wlkwneqcyu
   family: "R"
   gloss: "says"
-  short: "[E:tce pan] says [A:tce* skan] to [O:tce* ma]."
+  short: "[E:tce pan] says [A:tce* ecama] to [O:tce* ma]."
```

```suggestion
  short: "[E:tce pan] says [A:tce* man] to [O:tce* ma]."
```
They could say things in other languages, thus I think it's better to have "man" for now.